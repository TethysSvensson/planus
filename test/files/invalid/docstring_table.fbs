//! Okay
/// Okay
//! Bad1
/// Okay
table /// Bad2
/// Bad3
Foo /// Bad4
/// Bad5
{ /// Bad6
  //! Bad7
  /// Okay
  x /// Bad8
  /// Bad9
  : /// Bad10
  /// Bad11
  uint8 /// Bad12
  /// Bad13
  = /// Bad14
  /// Bad15
  1 /// Bad16
  /// Bad17
  ; /// Bad18
  //! Bad19
  /// Okay
  y /// Bad20
  /// Bad21
  : /// Bad22
  /// Bad23
  [ /// Bad24
  /// Bad25
  uint8 /// Bad26
  /// Bad27
  ] /// Bad28
  /// Bad29
  = /// Bad30
  /// Bad31
  [ /// Bad32
  /// Bad33
  ] /// Bad34
  /// Bad35
  ; /// Bad36
  //! Bad37
  /// Okay
  z /// Bad38
  /// Bad39
  : /// Bad40
  /// Bad41
  Foo /// Bad42
  /// Bad43
  ( /// Bad44
  /// Bad45
  required /// Bad46
  /// Bad47
  ) /// Bad48
  /// Bad49
  ; /// Bad50
/// Bad51
} /// Bad52
/// Bad53
//! Bad54
