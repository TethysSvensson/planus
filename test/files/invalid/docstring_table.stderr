error: Inner doc comments (those starting with `//!`) are only allowed at the beginning of the file and are used to document the namespace. If you meant to write a normal doc comment, those start with `///`.
  ┌─ test/files/invalid/docstring_table.fbs:3:1
  │
3 │ //! Bad1
  │ ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_table.fbs:5:7
  │
5 │ table /// Bad2
  │       ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_table.fbs:6:1
  │
6 │ /// Bad3
  │ ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_table.fbs:7:5
  │
7 │ Foo /// Bad4
  │     ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_table.fbs:8:1
  │
8 │ /// Bad5
  │ ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_table.fbs:9:3
  │
9 │ { /// Bad6
  │   ^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:57:1
   │
57 │ /// Bad51
   │ ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:58:3
   │
58 │ } /// Bad52
   │   ^^^^^^^^^

error: Inner doc comments (those starting with `//!`) are only allowed at the beginning of the file and are used to document the namespace. If you meant to write a normal doc comment, those start with `///`.
   ┌─ test/files/invalid/docstring_table.fbs:10:3
   │
10 │   //! Bad7
   │   ^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:12:5
   │
12 │   x /// Bad8
   │     ^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:13:3
   │
13 │   /// Bad9
   │   ^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:14:5
   │
14 │   : /// Bad10
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:15:3
   │
15 │   /// Bad11
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:16:9
   │
16 │   uint8 /// Bad12
   │         ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:17:3
   │
17 │   /// Bad13
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:18:5
   │
18 │   = /// Bad14
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:19:3
   │
19 │   /// Bad15
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:20:5
   │
20 │   1 /// Bad16
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:21:3
   │
21 │   /// Bad17
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:22:5
   │
22 │   ; /// Bad18
   │     ^^^^^^^^^

error: Inner doc comments (those starting with `//!`) are only allowed at the beginning of the file and are used to document the namespace. If you meant to write a normal doc comment, those start with `///`.
   ┌─ test/files/invalid/docstring_table.fbs:23:3
   │
23 │   //! Bad19
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:25:5
   │
25 │   y /// Bad20
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:26:3
   │
26 │   /// Bad21
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:27:5
   │
27 │   : /// Bad22
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:28:3
   │
28 │   /// Bad23
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:29:5
   │
29 │   [ /// Bad24
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:30:3
   │
30 │   /// Bad25
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:31:9
   │
31 │   uint8 /// Bad26
   │         ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:32:3
   │
32 │   /// Bad27
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:33:5
   │
33 │   ] /// Bad28
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:34:3
   │
34 │   /// Bad29
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:35:5
   │
35 │   = /// Bad30
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:36:3
   │
36 │   /// Bad31
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:37:5
   │
37 │   [ /// Bad32
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:38:3
   │
38 │   /// Bad33
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:39:5
   │
39 │   ] /// Bad34
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:40:3
   │
40 │   /// Bad35
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:41:5
   │
41 │   ; /// Bad36
   │     ^^^^^^^^^

error: Inner doc comments (those starting with `//!`) are only allowed at the beginning of the file and are used to document the namespace. If you meant to write a normal doc comment, those start with `///`.
   ┌─ test/files/invalid/docstring_table.fbs:42:3
   │
42 │   //! Bad37
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:44:5
   │
44 │   z /// Bad38
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:45:3
   │
45 │   /// Bad39
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:46:5
   │
46 │   : /// Bad40
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:47:3
   │
47 │   /// Bad41
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:48:7
   │
48 │   Foo /// Bad42
   │       ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:49:3
   │
49 │   /// Bad43
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:50:5
   │
50 │   ( /// Bad44
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:51:3
   │
51 │   /// Bad45
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:52:12
   │
52 │   required /// Bad46
   │            ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:53:3
   │
53 │   /// Bad47
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:54:5
   │
54 │   ) /// Bad48
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:55:3
   │
55 │   /// Bad49
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:56:5
   │
56 │   ; /// Bad50
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:59:1
   │
59 │ /// Bad53
   │ ^^^^^^^^^

error: Inner doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_table.fbs:60:1
   │
60 │ //! Bad54
   │ ^^^^^^^^^

